### PR TITLE
Move Language Picker loading into core

### PR DESCRIPTION
**Pull request by tomgreenfield**
In conjunction with https://github.com/adaptlearning/adapt_framework/pull/1253.

### CODE REVIEW

**Comment by moloko** [comment]
shouldn't there be a PR for spoor as well?

**Comment by brian-learningpool** [comment]
@tomgreenfield, should this PR be into the `feature/ml` branch rather than `master`?

File: js/languagePickerModel.js [hunk 1]
```diff
@@ -73,20 +47,6 @@ define([
 
       this.set('_languages', languages);
     },
```
Comment (moloko) [inline js/languagePickerModel.js]: you've got a trailing comma here

File: js/adapt-languagePicker.js [hunk 2]
```diff
@@ -42,20 +42,12 @@ define([
   function onOfflineStorageReady() {
     var previousLanguage = Adapt.offlineStorage.get("lang");
```
Comment (moloko) [inline js/adapt-languagePicker.js]: sorry I know I'm being sadly picky here but could this be changed to `storedLanguage`?

Comment (lc-thomasberger) [inline js/adapt-languagePicker.js]: yes, fine for me

**Pull request merged**